content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -47,6 +61,23 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 			return FileDataElementWrapper.getFromElement(this._domElement);
 		},
 
+		/**
+		 * Initializes object fields.
+		 */
+		_init : function() {
+
+			// Retrive file name
+			var fileInfoDom = this._domElement.getElementsByClassName(
+				FileElementWrapper.CLASS_FILE_INFO);
+
+			var fileTextSpan = fileInfoDom[0].getElementsByClassName(
+				FileElementWrapper.CLASS_SELECTABLE_TEXT);
+
+			var nameSrcDom = fileTextSpan.length === 0 ? fileInfoDom[0] : fileTextSpan[0];
+			this._fileName = nameSrcDom.getAttribute("title");
```

Comment:
Aha, I'll take a look this evening at that issue. If you have repro steps can you create an issue?